test/format/bulk.c

```diff
@@ -186,20 +190,24 @@ table_load(TABLE *base, TABLE *table)
             break;
         }
 
+        report_progress = (keyno < 5000 && keyno % 10 == 0) || keyno % 5000 == 0;
+        /* Report on progress. */
+        if (report_progress)
+            track(track_buf, keyno);
+
         /*
          * When first starting up, restart the enclosing transaction every 10 operations so we never
          * end up with an empty object. After 5K records, restart the transaction every 5K records
```

Comment:
This comment is now below and not clearly associated with the 5k usage in `report_progress`.

Reply:
[b39de35](https://github.com/wiredtiger/wiredtiger/pull/8496/commits/b39de35aeb45f988313485b9efc69c271ad13713)